assignment6/rearranging_cars_test.py

```diff
@@ -0,0 +1,26 @@
+'''
+Created on 07 July 2017
+
+@author: Eva
+'''
+
+import unittest
+from rearranging_cars import rearrange_cars, Move
+
+class UnknownLanguageTest(unittest.TestCase):
+    def setUp(self):
+        self.initial_array = [1, 2, 0, 3]
+        self.final_array = [3, 1, 2, 0]
+        self.solution = [Move(0,2), Move(3,0), Move(1,3), Move(2,1), Move(3,2)]
```

Comment:
From my opinion, it's better check, that returned sequence of moves leads you to the final state, because this is the point of the assignment. You can write helper function for this.